src/services/RennesApp.ts

```diff
@@ -47,6 +46,10 @@ export class RennesApp extends VcsApp {
     return this.get2DMap().olMap as Map
   }
 
+  getCurrentDistance() {
+    return this.get3DMap().getViewpointSync()?.cameraPosition[2]
```

Comment:
should it be `this.get3DMap().getViewpointSync()?.distance` since cameraPosition[2] is the z value of the camera? It might be the camera has a low z, but the distance is far